packages/ui/src/Rate/RateStars/index.tsx

```diff
@@ -27,7 +28,18 @@ export const MARKS_ARRAY = ['1', '2', '3', '4', '5'];
 const RateStars: ForwardRefExoticComponent<
   RateStarsProps & RefAttributes<HTMLDivElement>
 > = forwardRef(
-  ({value, onChange, className, size = 's', tooltipTrigger, ...props}, ref) => {
+  (
+    {
+      value,
+      onChange,
+      className,
+      starsClassName,
```

Comment:
на каждую звездочку, а stars как будто на контейнер
```suggestion
      starClassName,
```